### PR TITLE
Updating FL Studio to 24.1.1.4239

### PR DESCRIPTION
**Pull request by HevyMusic**
Deletion of 24.1.0.4225, adding of 24.1.1.4239. Updating the ProductCode

Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162576)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 886.)

File: manifests/i/ImageLine/FLStudio/24.1.1.4239/ImageLine.FLStudio.installer.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+PackageVersion: 24.1.1.4239
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallerSuccessCodes:
+- 1223
+ProductCode: FL Studio 24
+Installers:
+- Architecture: x64
+  InstallerUrl: https://install.image-line.com/flstudio/flstudio_win64_24.1.1.4239.exe
+  InstallerSha256: 905BCEDD2A15127E5FA6FC176B540FCDDC00B947CA07784956D1BFF6E4DE9FB0
```
Comment (stephengillie) [inline manifests/i/ImageLine/FLStudio/24.1.1.4239/ImageLine.FLStudio.installer.yaml:14]:
```suggestion
  InstallerSha256: B170B56785735EEAC067417CF8DE98A2D17CDE78C915D16EDD9FDBF97F85AF77
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> 7/11/2024 5:30:50 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   B170B56785735EEAC067417CF8DE98A2D17CDE78C915D16EDD9FDBF97F85AF77
Expected hash: 905bcedd2a15127e5fa6fc176b540fcddc00b947ca07784956d1bff6e4de9fb0

**Comment by HevyMusic** [comment]
Oups sorry, I got the email that the hash was incorrect, fixed it directly and pushed again.

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
I can't find the PR where these files were submitted, but unfortunately this manifest has already been added.

**Comment by stephengillie** [comment]
Close with reason: Merge Conflicts;

**Pull request closed without merging**